Ventilators/R4/resources/Questionnaire-R4-VentilatorsProgressNote.json

```diff
@@ -0,0 +1,1734 @@
+          "answerValueSet": "#yesno"
+        },
+        {
+          "linkId": "5.2",
+          "text": "Describe",
+          "type": "text"
+        }
+      ]
+    },
+    {
+      "linkId": "6",
+      "text": "Sujective",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "6.1",
+          "text": "Chief complaint / history of present illness and associated signs / symptoms",
+          "type": "text"
+        },
+        {
+          "linkId": "6.2",
+          "text": "Related past medical / surgical history",
+          "type": "text"
+        },
+        {
+          "linkId": "6.3",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "gtable"
+                  }
+                ]
+              }
+            }
+          ],
+          "text": "Medication",
+          "type": "group",
+          "repeats": true,
+          "item": [
+            {
+              "linkId": "6.3.1",
+              "text": "RxNorm",
+              "type": "string"
+            },
+            {
+              "linkId": "6.3.2",
+              "text": "Description",
+              "type": "string"
+            },
+            {
+              "linkId": "6.3.3",
+              "text": "Dose",
+              "type": "string"
+            },
+            {
+              "linkId": "6.3.4",
+              "text": "Frequency",
+              "type": "string"
+            },
+            {
+              "linkId": "6.3.5",
+              "text": "Route",
+              "type": "string"
+            },
+            {
+              "linkId": "6.3.6",
+              "text": "Status",
+              "type": "choice",
+              "answerOption": [
+                {
+                  "valueCoding": {
+                    "display": "New"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "display": "Current"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "display": "Modified"
+                  }
+                },
+                {
+                  "valueCoding": {
+                    "display": "Discountinued"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "linkId": "6.4",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"VentilatorsPrepopulation\".Allergies"
+              }
+            }
+          ],              
+          "text": "Allergy",
```

Comment:
I see you are populating allergy as open-choice which probably makes sense. I was trying to display as gtable but don't think it is possible to populate multiple items as the gtable is converted to a "group" and each of the item has different linkId. Even the items are displayed on the same row, they are separate items under the same group. Same as your RxNorm table. Thoughts?

Reply:
Since the Allergy table in relative simple (only two columns), I decide to combine them to open-choice.  In order to populate gtable, each item/column need to read separately. For example, if I use gtable here, then I will have one CQL for Allergy.code column and another CQL for Allergy.display column. And I will make sure that the order of these CQLs keeps consistent.